config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ ] Great job working on this milestone and submitting the correct project. It not professional to allow commented lines of code in our codebase. With the use of Git, we can always get a history of deleted codes. Kindly delete this line of commented code